content/codespaces/getting-started-with-codespaces/getting-started-with-your-python-project-in-codespaces.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Getting started with your Python project in Codespaces
+shortTitle: Getting started with your Python project
+intro: 'You can create a custom dev container with all the tools necessary to get started with your Python project in {% data variables.product.prodname_codespaces %}.'
+product: '{% data reusables.gated-features.codespaces %}'
+versions:
+  free-pro-team: '*'
+topics:
+  - Codespaces
+---
+
+
+### Introduction
```

Comment:
I want to ask about the header levels. From an accessibility standpoint shouldn't these start at level two (`##`)?

Reply:
I defer to our resident GitHub writer @timeyoutakeit for this! I'm working off a template so didn't change headers but Amy will know best :)

Reply:
We use H3 headings in our docs as standard.